db/seeds.rb

```
@@ -699,4 +771,66 @@
   ParticipationTerms.create!(
     terms: "Sample Participation Terms. Please update at '/admin/participation_terms'"
   )
+
+  #Create Team,TeamParticipant, ParticipantOrganizer and ChallengesOrganizer
+  organizer_ids = Organizer.all.ids
+  challenge_ids = Challenge.all.ids
+
+  challenge_ids.each do |challenge_id|
+    Team.create(name: "Team#{challenge_id}", challenge_id: challenge_id)
```

Please use methods with `!` to not fail silently e.g `create!` here.